packages/create-ponder/src/index.ts

```diff
@@ -426,10 +426,16 @@ export async function run({
     .help();
 
   // Check Nodejs version
-  if (Number(process.version.split(".")[0].slice(1)) < 18)
+  const _nodeVersion = process.version.split(".");
```

Comment:
Is there anywhere in `create-ponder` where we're generating the package.json (not just copying from templates) that needs to be updated?

Reply:
Good thought but no.